src/FINDOLOGIC/Export/XML/Page.php

```diff
@@ -60,6 +63,29 @@ public function getXml(): \DOMDocument
             $items->appendChild($itemDom);
         }
 
+        $this->validateWithSchema($document);
+
         return $document;
     }
+
+    /**
+     * Validates the export page against the schema. In case of violations, an exception is thrown.
+     *
+     * @param DOMDocument $document The document to validate.
+     */
+    private function validateWithSchema(DOMDocument $document): void
```

Comment:
Well done. Without the usage of any libraries. We stay as slim as possible.

Reply:
It was a little tricky to figure out how to capture the validation issues. Oh, the wonders of the PHP standard library...